depends/packages/qt.mk

```diff
@@ -31,7 +31,7 @@ $(package)_config_opts += -no-iconv
 $(package)_config_opts += -no-gif
 $(package)_config_opts += -no-freetype
 $(package)_config_opts += -no-nis
-$(package)_config_opts += -no-pch
+$(package)_config_opts += -pch
```

Comment:
It's from upstream switch to Trusty https://github.com/bitcoin/bitcoin/commit/564aaa2cd0ee057d9eedc8da1c900d90f43c89c6